lms/envs/common.py

```diff
@@ -212,9 +212,9 @@
     # when enrollment exceeds this number
     'MAX_ENROLLMENT_INSTR_BUTTONS': 200,
 
-    # Grade calculation started from the new instructor dashboard will write
-    # grades CSV files to S3 and give links for downloads.
-    'ENABLE_S3_GRADE_DOWNLOADS': False,
+    # Grade calculation started from the instructor dashboard will write grades
+    # CSV files to the configured storage backend and give links for downloads.
+    'ENABLE_GRADE_DOWNLOADS': False,
```

Comment:
The change of this feature flag will need the corresponding change in configuration along with a changelog entry.  @nedbat ideally we would make this backwards compatible in eclyptus and then fully deprecate the old flag in "F" release.  The danger with that is that we have to remember to clean up after ourselves in 6months.